### DISCUSSION TITLE
Call to undefined method Illuminate\Support\Facades\DB::extend()

### DISCUSSION THREAD

**Discussion by evsumantad** [jenssegers/laravel-mongodb#2009] [General]
- Laravel-mongodb Version: 3.2.3
- PHP Version: 7.2
- Database Driver & Version:

### Description:

Call to undefined method Illuminate\Support\Facades\DB::extend()
--
in MongodbServiceProvider.php line 2

### Steps to reproduce
1.
2.
3.

### Expected behaviour
Tell us what should happen

### Actual behaviour
Tell us what happens instead

<details><summary><b>Logs</b>:</summary>

Call to undefined method Illuminate\Support\Facades\DB::extend()
--
in MongodbServiceProvider.php line 28
at MongodbServiceProvider->Jenssegers\Mongodb\{closure}(object(DB), object(Application))in Container.php line 1019
at Container->fireCallbackArray(object(DB), array(object(Closure)))in Container.php line 965
at Container->fireResolvingCallbacks('Illuminate\\Support\\Facades\\DB', object(DB))in Container.php line 627
at Container->resolve('Illuminate\\Support\\Facades\\DB')in Container.php line 575
at Container->make('Illuminate\\Support\\Facades\\DB')in Application.php line 728
at Application->make('Illuminate\\Support\\Facades\\DB')in Container.php line 862
at Container->resolveClass(object(ReflectionParameter))in Container.php line 790
at Container->resolveDependencies(array(object(ReflectionParameter), object(ReflectionParameter), object(ReflectionParameter), object(ReflectionParameter), object(ReflectionParameter), object(ReflectionParameter), object(ReflectionParameter), object(ReflectionParameter), object(ReflectionParameter), object(ReflectionParameter), object(ReflectionParameter), object(ReflectionParameter), object(ReflectionParameter), object(ReflectionParameter), object(ReflectionParameter)))in Container.php line 757
at Container->build('App\\Http\\Controllers\\HomeController')in Container.php line 608
at Container->resolve('App\\Http\\Controllers\\HomeController')in Container.php line 575
at Container->make('App\\Http\\Controllers\\HomeController')in Application.php line 728
at Application->make('App\\Http\\Controllers\\HomeController')in Route.php line 217
at Route->getController()in Route.php line 747
at Route->controllerMiddleware()in Route.php line 708
at Route->gatherMiddleware()in Router.php line 585
at Router->gatherRouteMiddleware(object(Route))in Router.php line 565
at Router->runRouteWithinStack(object(Route), object(Request))in Router.php line 533
at Router->dispatchToRoute(object(Request))in Router.php line 511
at Router->dispatch(object(Request))in Kernel.php line 176
at Kernel->Illuminate\Foundation\Http\{closure}(object(Request))in Pipeline.php line 30
at Pipeline->Illuminate\Routing\{closure}(object(Request))in Debugbar.php line 51
at Debugbar->handle(object(Request), object(Closure))in Pipeline.php line 148
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))in Pipeline.php line 53
at Pipeline->Illuminate\Routing\{closure}(object(Request))in TransformsRequest.php line 30
at TransformsRequest->handle(object(Request), object(Closure))in Pipeline.php line 148
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))in Pipeline.php line 53
at Pipeline->Illuminate\Routing\{closure}(object(Request))in Currencycheck.php line 50
at Currencycheck->handle(object(Request), object(Closure))in Pipeline.php line 148
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))in Pipeline.php line 53
at Pipeline->Illuminate\Routing\{closure}(object(Request))in TransformsRequest.php line 30
at TransformsRequest->handle(object(Request), object(Closure))in Pipeline.php line 148
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))in Pipeline.php line 53
at Pipeline->Illuminate\Routing\{closure}(object(Request))in ValidatePostSize.php line 27
at ValidatePostSize->handle(object(Request), object(Closure))in Pipeline.php line 148
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))in Pipeline.php line 53
at Pipeline->Illuminate\Routing\{closure}(object(Request))in CheckForMaintenanceMode.php line 46
at CheckForMaintenanceMode->handle(object(Request), object(Closure))in Pipeline.php line 148
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))in Pipeline.php line 53
at Pipeline->Illuminate\Routing\{closure}(object(Request))in Pipeline.php line 102
at Pipeline->then(object(Closure))in Kernel.php line 151
at Kernel->sendRequestThroughRouter(object(Request))in Kernel.php line 116
at Kernel->handle(object(Request))in index.php line 53

</details>

**Comment by Smolevich** [comment]
@sumantafinarb, how you can configure laravel-mongodb?

**Comment by evsumantad** [comment]
Yes by using you library and

**Comment by Smolevich** [comment]
Show configuration please
and simple case for investigation
